configs/paths.go

```diff
@@ -0,0 +1,5 @@
+package configs
+
+const (
+	ValidatorDir = "validator-data"
```

Comment:
I prefer to use the new one `paths.go` file because `message.go` is huge and confusing, at least for me. If you agree, I can move `ExecutionDefaultDataDir`, `ConsensusDefaultDataDir` and `ConsensusDefaultDataDir` to `paths.go` and rename them to `ExecutionDir`, `ConsensusDir` and `ValidatorDir` respectively.

Reply:
The problem I see with those `*DefaulDataDir` is the `./` within them. In this case I prefer if all the path constants are in `paths.go`. Do you think it is a good idea to rename the `*DefaulDataDir` to `*Dir` as you suggested and update the code that uses the actual `ValidatorDir` to stripe the `./` prefix from the path? This is the first solution I came with, I'm open to an alternative approach.

Reply:
I think `./` depends on the context where it is used, I prefer to remove `./` from the `*Dir` values and add it where it is needed

Reply:
Done c37313ccfaff6811c980c1a8960967a6ca729291, 382f393c544f611dc718b23de2b90e53a761a597, be9de0b2a5fdc95255ddbdb552e97c1cb0e20440, 8c71fe227d628f3aefb38559ac2b65c86712546b